### PR TITLE
Annotations - ready to merge

### PR DESCRIPTION
**Pull request by jackbackes**

### CODE REVIEW

**Comment by jdhang** [comment]
Testing comment

File: browser/js/components/Annotator/actions.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+'use strict'
```
Comment (jdhang) [inline browser/js/components/Annotator/actions.js:1]: testing comment on file

**Comment by jackbackes** [comment]
@jdhang @jancodes @jes708 any comments on this?

**Comment by jancodes** [comment]
👍  merge request because of action tree?

**Pull request merged**